argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc

```diff
@@ -36,6 +36,15 @@ include::modules/gitops-creating-rolloutmanager-custom-resource.adoc[leveloffset
 // Deleting a RolloutManager custom resource
 include::modules/gitops-deleting-rolloutmanager-custom-resource.adoc[leveloffset=+1]
 
+// Argo Rollouts CLI overview
+include::modules/gitops-argo-rollouts-cli-overview.adoc[leveloffset=+1]
+
+// Installing Argo Rollouts CLI by using Linux
```

Comment:
I am thinking to maybe call these chapters: `Installing Argo Rollouts CLI on Linux` and `Installing Argo Rollouts CLI on Mac OS` as it sounds better and is more concise.

Reply:
Fixed.